lms/envs/common.py

```diff
@@ -3270,6 +3270,7 @@
 ENTERPRISE_PUBLIC_ENROLLMENT_API_URL = ENTERPRISE_ENROLLMENT_API_URL
 ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES = ['audit', 'honor']
 ENTERPRISE_SUPPORT_URL = ''
+ENTERPRISE_REPORTING_SECRET = ''
```

Comment:
@brittneyexline  should this default also be 16 characters long as it is in configuration?

Reply:
good point, just updated.